anycloud/docs/topics/architecture.md

```diff
@@ -0,0 +1,12 @@
+AnyCloud is built for performance on the [Rust](https://rust-lang.org) and [Alan](https://alan-lang.org) programming languages. It is accessed via a CLI that takes cloud provider credentials and a `Dockerfile` with a HTTP server listening on port `8088` as input. Your container/server is deployed to virtual machines in the account with the specified credentials and runs with a sidecar process that proxies requests into the HTTP server. 
+
+
+<img src="../assets/architecture.png" width=500 />
```

Comment:
Nit on the picture: should it have the Alan logo in the sidecar process box?

Reply:
It is not recognizable, yet, so I'd rather keep the image more minimalistic for now